scripts/mocks.mockgen.txt

```diff
@@ -40,5 +40,7 @@ github.com/ava-labs/avalanchego/vms/proposervm=PostForkBlock=vms/proposervm/mock
 github.com/ava-labs/avalanchego/vms/registry=VMGetter=vms/registry/mock_vm_getter.go
 github.com/ava-labs/avalanchego/vms/registry=VMRegistry=vms/registry/mock_vm_registry.go
 github.com/ava-labs/avalanchego/vms=Factory,Manager=vms/mock_manager.go
+github.com/ava-labs/avalanchego/wallet/subnet/primary/common=ChainUTXOs=wallet/subnet/primary/common/mock_chain_utxos.go
```

Comment:
Note: I tried to avoid introducing mocks in the wallet but currently could not.
As it is now, P-chain builder and signer testing runs into circular dependencies issues if we try and do it without mocks.
This is something we should try and improve, if we want the wallet to be the one way to build transactions, for users, in e2e tests and in `platformvm` and `avm` package UTs as well.
We'll get there in time

Reply:
I removed the mocks